.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+on: [push]
```

Comment:
```suggestion
on: [push, pull_request]
```

Reply:
I was about to say, well I guess that's why it wasn't running on this PR, but it's still not running :sweat_smile:

Reply:
cherry-picked (after fixing whitespace issues) into main; see if that kicks things of ?